common/display/virtualdisplay.h

```diff
@@ -41,6 +41,10 @@ class VirtualDisplay : public NativeDisplay {
 
   bool SetActiveConfig(uint32_t config) override;
 
+  bool SetResolution(const HwcRect<int32_t>& ) override {
+    return false;
```

Comment:
We could have a default implementation in NativeDisplay returning false and we can avoid overrides in multiple inherited classes here

Reply:
Yes, will do that. Multiple inherited classes may be extended as needed in future.